native/winp.h

```diff
@@ -1,18 +1,26 @@
 #pragma once
 #include "stdafx.h"
 
-BOOL WINAPI KillProcessEx(IN DWORD dwProcessId, IN BOOL bTree);
-
+// Sets bit 29 in order to keep the codes in the user space
+#define reportErrorWithCode(env,code,msg)	SetLastError(code + 0x10000000); error(env,__FILE__,__LINE__,msg);

```

Comment:
Nope, because it is a native stuff. Ideally I should create enum there as well, but I firstly need to think about fatal/not-fatal errors (E.g. Process memory block unavailability is not fatal, and it is one of potential JENKINS-30782 root causes)